runtime/compiler/control/JITServerCompilationThread.cpp

```diff
@@ -122,3 +235,95 @@ TR::CompilationInfoPerThreadRemote::cacheResolvedMirrorMethodsPersistIPInfo(TR_R
 
    _resolvedMirrorMethodsPersistIPInfo->push_back(resolvedMethod);
    }
+
+/**
+ * Code to be executed on the JITServer to cache NULL value static class
+ *
+ * @param ramClass The static class of interest as part of the key
+ * @param cpIndex The constant pool index of interest as part of the key
+ * @return void
+ */
+void
+TR::CompilationInfoPerThreadRemote::cacheNullClassOfStatic(TR_OpaqueClassBlock *ramClass, int32_t cpIndex)
+   {
+   TR_OpaqueClassBlock *nullClazz = NULL;
+   cacheToPerCompilationMap(_classOfStaticMap, std::make_pair(ramClass, cpIndex), nullClazz);
+   }
+
+/**
+ * Code to be executed on the JITServer to retrieve NULL value static class 
```

Comment:
@brief Method executed by JITServer to determine if a previous `classOfStatic()` query returned NULL